common/src/main/java/io/netty/util/concurrent/SingleThreadEventExecutor.java

```diff
@@ -254,22 +249,14 @@ protected Runnable takeTask() {
     }
 
     private void fetchFromDelayedQueue() {
```

Comment:
This method is the same as `GlobalEventExecutor. fetchFromDelayedQueue`...can we share?